hnn_core/basket.py

```diff
@@ -149,20 +149,14 @@ def parreceive_ext(self, type, gid, gid_dict, pos_dict, p_ext):
 
                 # connections depend on location of input - why only
                 # for L2 basket and not L5 basket?
-                if p_ext['loc'] == 'proximal':
+                if p_ext['loc'] in ['proximal', 'distal']:
```

Comment:
This is fair, in fact I was looking for something that already existed but couldn't find it. Are we meeting tomorrow afternoon? Should we try to implement this exact thing with pair/triplet programming? I want to keep the discussion more concrete and less "meta". 

I also tagged you in a follow up PR I planned. Should I just append that PR on to this one (it's a fairly straightforward change)? It will make it easier for me to make changes on top without worrying about rebase conflicts. Or would you prefer to review it separately?

Reply:
I see, so https://github.com/jasmainak/hnn-core/pull/1 is basically just moving `parreceive_` methods up to their parent classes. I think this is manageable to merge and review in this PR.

What time are we meeting tomorrow? I'm down for another programming session, though I'm also happy to attempt implementing my proposed changes asynchronously if that would be helpful.

Reply:
okay I copied what was in #https://github.com/jasmainak/hnn-core/pull/1 into this PR then.